src/notation/internal/notationactioncontroller.cpp

```diff
@@ -68,13 +68,13 @@ void NotationActionController::init()
     dispatcher()->reg(this, "pad-dot4", [this]() { padNote(Pad::DOT4); });
     dispatcher()->reg(this, "pad-rest", [this]() { padNote(Pad::REST); });
 
-    dispatcher()->reg(this, "note-c", [this]() { addNote(NoteName::C, NoteAddingMode::NextChord); });
-    dispatcher()->reg(this, "note-d", [this]() { addNote(NoteName::D, NoteAddingMode::NextChord); });
-    dispatcher()->reg(this, "note-e", [this]() { addNote(NoteName::E, NoteAddingMode::NextChord); });
-    dispatcher()->reg(this, "note-f", [this]() { addNote(NoteName::F, NoteAddingMode::NextChord); });
-    dispatcher()->reg(this, "note-g", [this]() { addNote(NoteName::G, NoteAddingMode::NextChord); });
-    dispatcher()->reg(this, "note-a", [this]() { addNote(NoteName::A, NoteAddingMode::NextChord); });
-    dispatcher()->reg(this, "note-b", [this]() { addNote(NoteName::B, NoteAddingMode::NextChord); });
+    registerAction("note-c", [this]() { addNote(NoteName::C, NoteAddingMode::NextChord); }, &NotationActionController::isStandardStaff);
+    registerAction("note-d", [this]() { addNote(NoteName::D, NoteAddingMode::NextChord); }, &NotationActionController::isStandardStaff);
+    registerAction("note-e", [this]() { addNote(NoteName::E, NoteAddingMode::NextChord); }, &NotationActionController::isStandardStaff);
+    registerAction("note-f", [this]() { addNote(NoteName::F, NoteAddingMode::NextChord); }, &NotationActionController::isStandardStaff);
+    registerAction("note-g", [this]() { addNote(NoteName::G, NoteAddingMode::NextChord); }, &NotationActionController::isStandardStaff);
+    registerAction("note-a", [this]() { addNote(NoteName::A, NoteAddingMode::NextChord); }, &NotationActionController::isStandardStaff);
+    registerAction("note-b", [this]() { addNote(NoteName::B, NoteAddingMode::NextChord); }, &NotationActionController::isStandardStaff);
```

Comment:
Could probably be done in a loop too - except that the NoteName enum increases from C-B whereas the char codes for c - b don't, and C++ doesn't allow indexing of enum values that I'm aware of.  But there is a lot of repetition here...

Reply:
> registerNoteAction('a', NoteName::A);
Good idea, we can just add a lambda ...